pdns/recursordist/rec_channel_rec.cc

```diff
@@ -2085,7 +2104,8 @@ static RecursorControlChannel::Answer help()
           "reload-lua-config [filename]     (re)load Lua configuration file\n"
           "reload-zones                     reload all auth and forward zones\n"
           "set-ecs-minimum-ttl value        set ecs-minimum-ttl-override\n"
-          "set-max-cache-entries value      set new maximum cache size\n"
+          "set-max-aggr-nsec-cache-size     set new maximum aggressive NSEC cache size\n"
```

Comment:
```suggestion
          "set-max-aggr-nsec-cache-size value set new maximum aggressive NSEC cache size\n"
```
Although it means re-aligning the existing entries, I'm afraid :-/